src/libraries/tests.proj

```diff
@@ -621,15 +621,22 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(ArchiveTests)' == 'true' and '$(TargetOS)' == 'ios' and '$(IsManualOrRollingBuild)' == 'true'">
+    <ProjectReference Include="$(MonoProjectRoot)sample\iOS\Program.csproj"
```

Comment:
This technically enables the test for devices.  Should we do that in https://github.com/dotnet/runtime/pull/86089 instead?